test/navigation/NavigatorSpec.scala

```diff
@@ -76,6 +76,18 @@ class NavigatorSpec extends BaseSpec with ScalaCheckPropertyChecks {
         }
       }
     }
+    "check active  bank account page" - {
```

Comment:
The page is active bank account page not the check active bank account page, also remove additional space